src/pages/index.tsx

```diff
@@ -15,21 +18,51 @@ import {
 } from '@/components/constants/task-status';
 
 const TASKS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/tasks`;
+const SELF_URL = 'https://api.realdevsquad.com/users/self';
```

Comment:
Why not use `NEXT_PUBLIC_BASE_URL` here?

Reply:
The NEXT_PUBLIC_BASE_URL is the staging url and has no roles with admin or super_user. The staging url only has member roles defined there